driver/flambda_backend_args.ml

```diff
@@ -495,6 +495,15 @@ let mk_no_dwarf_for_startup_file f =
   "-gno-startup", Arg.Unit f, " Emit the same DWARF information for the\n\
     \     startup file as the upstream compiler"
 
+let mk_use_cached_generic_functions f =
+  "-use-cached-generic-functions", Arg.Unit f, " Use the cached startup"
+;;
+
+let mk_cached_generic_functions_path f =
+  "-cached-generic-functions-path", Arg.String f,
+  "<file>  Set the path of the cached startup (default to cache-startup.o)"
```

Comment:
The default name does not match `Flambda_backend_flags.cached_generic_functions_path`.

Reply:
Fixed in the next commit